documentation/kubernetes-upgrade.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Upgrading kubeadm clusters from v1.7/v1.8 to v1.9 with vSphere Cloud Provider
```

Comment:
What is kubeadm clusters? There is no kubeadm cluster. Did you mean to say Kubernetes cluster deployed via kubeadm? If so, please correct it.

Reply:
I meant to say - Kubernetes cluster deployed via kubeadm. I will update it.